src/juno-node.ts

```diff
@@ -1,3 +1,5 @@
+import * as net from 'net';
+import * as fs from 'fs';
```

Comment:
Same goes here:
`import { lstat } from 'fs/promises';`

Reply:
The support for `fs/promises` alias is added for Node v14. <br>
Ref: [https://github.com/nodejs/node/pull/31553](https://github.com/nodejs/node/pull/31553)
Version Release Ref: [https://github.com/nodejs/node/pull/32181](https://github.com/nodejs/node/pull/32181)

I don't see a better method than, <br>
```javascript
import { promises as fs } from 'fs';
const { lstat } = fs;
```

But Rollup statically analyzes the code being imported, and will exclude anything that isn't actually used. So, this solution should work. Any suggestion?

Reply:
`fs-promises` instead of `fs`. Makes it easier for someone new reading the code to understand what's happening. Otherwise, go ahead with it

Reply:
I'll have to add `fs-promises` (another module) as dependency. It seems more overhead than declaring promises variable. I'll continue with `fs` module.

Reply:
No, I mean declare the variable as `fs-promises`. I was referring to the variable name.

Now that it hits me, use `fsPromises`. The variable name matters.

Reply:
Oh xD. Adding another module to dependency is cool?

Reply:
I'd prefer not to, unless it's required. As the people who make a library, I'd rather not unnecessarily bloat our end user's npm dependencies. If you're using something all over the place, then it makes sense. In this specific case, for a one-time need, however, I'd rather not.

Reply:
Okayy